src/stores/upload.store.tsx

```diff
@@ -151,25 +322,59 @@ ${dbfFile.meta.keys.map((k, i) => `${k} : ${titleColumn[i]}`).join("\n")}`;
     });
   };
 
+  const generateCsvTxt = () => {
```

Comment:
This function is completly duplicate of `generateTxt()` please don't just copy paste functions try to add conditional checks everywhere and avoid code duplicacy